dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -334,6 +334,12 @@
 			interrupt-parent = <&ace_intc>;
 		};
 
+		alh0: alh1: alh@24400 {
```

Comment:
why node label for both `alh0` and `alh1`?

Reply:
can you copy the comment from intel_adsp_cavs25.dtsi

Reply:
yes, done